### PR TITLE
[WIP] Stepping: Continue to here

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: #3503 

I really don't know why but the debugger doesn't stop at the hidden breakpoint.
Maybe someone of the core team can pair up with me so I can solve this...

### CODE REVIEW

File: src/actions/pause.js [hunk 1]
```diff
@@ -46,6 +46,21 @@ export function resumed() {
...
 
+export function continueToHere(line: number) {
+  return async function({ dispatch, getState, client, sourceMaps }: ThunkArgs) {
+    const source = getSelectedSource(getState()).toJS();
+
+    await dispatch(
+      addHiddenBreakpoint({
+        line,
+        column: null,
+        sourceId: source.id
```
Comment (jasonLaster) [inline src/actions/pause.js]: Try using undefined instead of null.

File: src/components/Editor/GutterMenu.js [hunk 2]
```diff
@@ -65,7 +71,17 @@ export default function GutterMenu({
       : gutterItems.addConditional
   );
 
-  const items = [toggleBreakpointItem, conditionalBreakpoint];
+  let items = [toggleBreakpointItem, conditionalBreakpoint];
+
+  if (pauseData) {
+    const continueToHereItem = {
+      accesskey: "C",
+      disabled: false,
```
Comment (jasonLaster) [inline src/components/Editor/GutterMenu.js]: The access key should be in the properties file as well

File: src/actions/pause.js [hunk 3]
```diff
@@ -46,6 +46,21 @@ export function resumed() {
   };
 }
 
+export function continueToHere(line: number) {
+  return async function({ dispatch, getState, client, sourceMaps }: ThunkArgs) {
+    const source = getSelectedSource(getState()).toJS();
+
+    await dispatch(
+      addHiddenBreakpoint({
+        line,
```
Comment (jasonLaster) [inline src/actions/pause.js]: If the line number is off by one, check what we do in actions/addBreakpoint

File: assets/panel/debugger.properties [hunk 4]
```diff
@@ -294,6 +294,10 @@ editor.searchResults.prevResult=Previous Result
 # toggling search type buttons(function search, variable search)
 editor.searchTypeToggleTitle=Search for:
 
+# LOCALIZATION NOTE (editor.continueToHere): Editor gutter context
+# menu item for jumping to a new paused location
+editor.continueToHere=Continue To Here
```
Comment (flodolo) [inline assets/panel/debugger.properties]: I suggest to use .label for labels.

```
# LOCALIZATION NOTE (editor.continueToHere.label): Editor gutter context
# menu item for jumping to a new paused location
editor.continueToHere.label=Continue To Here
editor.continueToHere.accesskey=C
```

**Comment by nyrosmith** [comment]
@flodolo pls review again. I made changes accoring to your suggestions.
NOTE: I changed the accesskey to "H" because "C" is already used by conditionalBreakpoint

**Comment by nyrosmith** [comment]
continuetohere

**Pull request merged**